src/Corrade/Utility/Debug.cpp

```diff
@@ -42,6 +49,46 @@ template<> inline void toStream<Implementation::DebugOstreamFallback>(std::ostre
     value.apply(s);
 }
 
+#ifdef CORRADE_TARGET_WINDOWS
+HANDLE getStdHandle(const std::ostream* s) {
+    if(!s) return INVALID_HANDLE_VALUE;
```

Comment:
You're right. The null-pointer case is implictitly handled. But sometimes I prefer expliciticity ;)